logging/logadmin/logadmin.go

```diff
@@ -243,9 +243,24 @@ func listLogEntriesRequest(parent string, opts []EntriesOption) *logpb.ListLogEn
 	for _, opt := range opts {
 		opt.set(req)
 	}
+	req.Filter = defaultTimestamp(req.Filter)
 	return req
 }
 
+func defaultTimestamp(filter string) string {
+	dayAgo := time.Now().Add(-24 * time.Hour).UTC()
```

Comment:
Confirming - is the semantic in gcloud to look at the last 24 hours in UTC time zone or current time zone?

Reply:
It should be current timezone to avoid unforeseen edge cases.  I'll leave this logic as is